examples/core/Graphormer/train.py

```diff
@@ -0,0 +1,493 @@
+import argparse
+import random
+
+import torch as th
+import torch.nn as nn
+import torch.nn.functional as F
+from accelerate import Accelerator
+from dgl import shortest_dist
+from dgl.data import download
+from dgl.dataloading import GraphDataLoader
+from dgl.nn import DegreeEncoder, GraphormerLayer, PathEncoder, SpatialEncoder
+from ogb.graphproppred import DglGraphPropPredDataset, Evaluator
+from torch.nn.utils.rnn import pad_sequence
+from transformers.optimization import (
+    AdamW,
+    get_polynomial_decay_schedule_with_warmup,
+)
+
+
+class Graphormer(nn.Module):
+    def __init__(
+        self,
+        num_classes=1,
+        edge_dim=3,
+        num_atoms=4608,
+        max_degree=512,
+        num_spatial=511,
+        multi_hop_max_dist=5,
+        num_encoder_layers=12,
+        embedding_dim=768,
+        ffn_embedding_dim=768,
+        num_attention_heads=32,
+        dropout=0.1,
+        pre_layernorm=True,
+        activation_fn=nn.GELU(),
+    ):
+
+        super().__init__()
+        self.dropout = nn.Dropout(p=dropout)
+        self.embedding_dim = embedding_dim
+        self.num_heads = num_attention_heads
+
+        # 1 for graph token
```

Comment:
What does "1 for graph token" mean? Can you be more specific?

Reply:
I am referring to L47: `self.graph_token = nn.Embedding(1, embedding_dim)`. Is the comment on L43 necessary?